language/Russian/strings.po

```diff
@@ -5899,11 +5961,11 @@ msgstr ""
 #: plugins/cheatfind/init.lua:751
 #, lua-format
 msgid "Use this if you want to poke %s"
-msgstr ""
+msgstr "Используется для указания %s"
```

Comment:
“Poke” here refers to replacing a value in memory.  Is this translation appropriate?